### ISSUE TITLE
Any way to parse and render ejs tags from DB?

### ISSUE THREAD

**Issue by alexandros-bantzos** [mde/ejs#484]
Hello,

I tried to search about the following issue and even made a few tests but no luck.

Is there any way to parse ejs tags from DB and render them afterwards?

an example is like:

EJS Template:
`<p>This is a text with <%- dataFromDB %></p>`

dataFromDB:
`<a href="#"><%= aGlobalVariable %></a>`

**Comment by mde** [comment]
I don't understand your question. You can store template text anywhere you like: in a text file, in a DB, anywhere. Just pull the string of text out, and use it as your template.

**Comment by alexandros-bantzos** [comment]
No, my question is, if I can inject an ejs tag from the database, into the template and make it render it.

Another example:
```
<div>
	<div><%- htmlBody %></div>
</div>
```

Where `htmlBody` contains some html but also contains another ejs tag, like:
`<......some-html......><div><%= someGlobalVariable %></div><......some-html......>`

**TLDR;**
Can I do this? → `<%- "<div><%= 'output this' %></div>" %>`

**Comment by mde** [comment]
Sounds like you're wanting to use EJS code to generate EJS templates. This is a bit weird, but possible. You can use a custom delimiter for either the template text you've stored, or the outer template:

```javascript
> let ejs = require('./ejs');
undefined
ejs.render('<@= data @><% this tag will remain as-is %>', {data: 'howdy'}, {delimiter: '@'});
'howdy<% this tag will remain as-is %>'
```
